src/main/java/net/glowstone/io/anvil/RegionFile.java

```diff
@@ -194,20 +191,14 @@ public RegionFile(File path) throws IOException {
             int numSectors = offset & 255;
 
             if (offset != 0 && startSector >= 0 && startSector + numSectors <= totalSectors) {
-                for (int sectorNum = 0; sectorNum < numSectors; ++sectorNum) {
-                    sectorsUsed.set(startSector + sectorNum);
-                }
+                sectorsUsed.set(startSector, startSector + numSectors + 1);
             } else if (offset != 0) {
                 GlowServer.logger.warning(
                         "Region \"" + path + "\": offsets[" + i + "] = " + offset + " -> "
                                 + startSector
                                 + "," + numSectors + " does not fit");
             }
         }
-        // read timestamps from timestamp table
```

Comment:
Although they are not used, they are part of the spec. Could you change usage of timestamps to be a configurable option?